src/DateChanger.tsx

```diff
@@ -117,20 +113,8 @@ export function DateChanger({
             )}
             value={dateState}
             onChange={(e) => setDateState(e.target.value)}
+            onBlur={() => onSave(dateState)}
```

Comment:
Using the onBlur attribute seems like a great idea, however I think I found two edge cases, we might have to think about:
- If the field is not "touched" - i.e. if the user only edits it using the edit button, but does not actually change the date, clicking outside does not de-select the field anymore...
- What happens if I include this component / the formik wrapper `FormikDateChanger` in some form and change to a different step of the form without clicking next to the field before? Is the state still updated or are those changes lost?

Reply:
It seems like the second edge case works - i.e. if I change the date and then switch to a different page / component / ..., the onBlur method seems to be called before the component is unmounted - at least in the ladle design-system framework.

Reply:
First edge case has been resolve with a `useEffect`, which ensures that the field is immediately in focus on edit.